chrome/content/zotfile/zotfile.js

```diff
@@ -1926,7 +1926,14 @@ Zotero.ZotFile = {
               // create a nslFile Object of the destination folder
               var dir = this.createFile(destination);
               // move file to new location
-              file.moveTo(dir, filename);
+              if(destination.trim() == "/") { 
+              	//if there's no destination, then the file will be renamed in-place
+              	alert("You have not set the location to move the file. File will only be renamed.");
```

Comment:
Surely I can use. I didn't know well enough. I'll look and make necessary changes.